src/ResourceManager/Sql/Commands.Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -101,6 +100,11 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase
         [Alias("Tag")]
         public Hashtable Tags { get; set; }
 
+        [Parameter(Mandatory = false,
+            HelpMessage = "The name of the sample schema to apply when creating this database.")]
+        [ValidateNotNullOrEmpty]
+        public string SampleName { get; set; }
```

Comment:
@jaredmoo do you think we should add the `ValidateSet` attribute to this parameter so that only specific values can be passed through? You can always add more values later when more samples are created. This will prevent users from putting in `Foo` and getting an error saying that the sample `Foo` does not exist and allow them to tab through the possible samples that are offered.